scripts/agent_automation.py

```
@@ -256,9 +256,7 @@ def status_monitor(self):
                 tasks = self.taskmaster.get_task_list()
 
                 print(f"\n{'='*60}")
```

Ruff (0.11.9)

`print` found

Remove `print`

(T201)